src/go/rpk/pkg/cli/version/versioncmd/versioncmd.go

```diff
@@ -106,7 +106,7 @@ To get only the rpk version, use 'rpk --version'.`,
 }
 
 func printRpkVersion(rv rpkVersion) {
-	fmt.Printf(`Version:     %s
+	fmt.Printf(`Rpk version: %s
```

Comment:
```suggestion
	fmt.Printf(`rpk version: %s
```

I think it's a product directive to always use `rpk` in lowercase

Reply:
yes, correct!